src/structures.rs

```diff
@@ -18,13 +20,16 @@ struct Connection {
     arr_stop: u32,
 }
 
-struct Footpath {}
+struct Footpath {
+    to: u32,
+    duration: u16,
```

Comment:
u16 for duration will be small, no?

Reply:
no, everything is in minutes, not seconds. Maybe I’ll regret that later, but for now I’ll just ignore